ClientApp/viking-vault/src/Common/Toggle.tsx

```diff
@@ -0,0 +1,19 @@
+import * as React from "react";
+import './Toggle.css';
+import { constants } from "../Resources/Constants";
+
+interface IToggleState{
+    toggleSwitch: any
+}
+
+function Toggle (props: IToggleState)
+{
```

Comment:
JavaScript convention is that the first code block is on the same line
blah {
}